app/Http/Middleware/RequireTwoFactorAuthentication.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
```

Comment:
All files belonging to the project that aren't Laravel specific should have the license header, should be able to just copy it out of an existing one. 👍

Reply:
Will do.